test/spell-checker.js

```diff
@@ -78,6 +78,13 @@ eslintTester.addRuleTest('rules/spell-checker', {
             errors: [
                 { message: 'You have a misspelled word: tsih on String'},
                 { message: 'You have a misspelled word: srting on String'}]
+        },
+        {
+            code: 'var a = 1 // color is a comment normalize dict',
+            args:[2, {lang: 'en_GB', skipWords: ['dict']}],
+            errors: [
+                { message: 'You have a misspelled word: color on Comment'},
+                { message: 'You have a misspelled word: normalize on Comment'}]
```

Comment:
I tested locally and normalize is not failing in my local test. Could you please correct and find another word not present in the US dict ?
And please add another test case for the British happy path with British words and the test not failing. 
Something like 
````
 code: 'var cheque = 1 // cheque is in colour tsih is a comment dict',
             args:[2, {lang: 'en_GB', skipWords: ['dict']}],
             errors: [
                 { message: 'You have a misspelled word: tsih on Comment'}]
```
thanks for the PR!